entrypoint.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 python manage.py makemigrations
 python manage.py migrate --noinput
+python manage.py update_index
```

Comment:
With this entrypoint.sh removed from `.dockerfile` do we have other methods of triggering update_index? I cannot see any other reference to it, and search may be broken if an index is never built.

Reply:
@akmiller01 

`update_index` is run on deployment via the dokku `predeploy` script in `app.json`:

https://github.com/IATI/IATI-Standard-Website/blob/feature/elasticsearch/app.json#L6

Locally, `update_index` can be run manually using:

`docker-compose run web python ./manage.py update_index`

Reply:
Yep. That works locally but I'm afraid Travis needs an extra bit of config in order to do it for us.

Reply:
Aha - is that a big task? 

I'm not that familiar with Travis config, so not sure if that's really tricky or not.

One of the reasons I took this out of the entrypoint was due to a race condition, i.e. ES not fully up and running before the `update_index` command was called, so I think that probably needs to be considered also. 

Apologies if this has caused more issues than it's solved!